zebra-rpc/src/methods/get_block_template_rpcs/types/get_block_template.rs

```diff
@@ -283,33 +286,6 @@ impl GetBlockTemplate {
     }
 }
 
-/// Error response to a `getblocktemplate` RPC request in proposal mode.
-#[derive(Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
-#[serde(rename_all = "kebab-case")]
-pub enum ProposalRejectReason {
-    /// Block proposal rejected as invalid.
-    Rejected,
-}
```

Comment:
Just a note for next time: please try to avoid large changes in a PR when someone has based another PR off it. It makes merging any changes harder.

There's also no need to backport changes that will happen in the next PR.

Reply:
Noted. I should've checked those files had been changed in your PR, my bad here.